nextcloudappstore/core/models.py

```diff
@@ -39,6 +39,10 @@ class App(TranslatableModel):
                                             verbose_name=_('Co-Maintainers'),
                                             related_name='co_maintained_apps',
                                             blank=True)
+    recommendations = models.ManyToManyField('auth.User',
```

Comment:
We should user `settings.AUTH_USER_MODEL` for ForeigKeys and ManyToManyfields.
See https://docs.djangoproject.com/en/1.9/topics/auth/customizing/#referencing-the-user-model